frontend/public/components/resource-quota.jsx

```diff
@@ -13,11 +15,29 @@ const menuActions = [...common];
 
 const quotaKind = quota => quota.metadata.namespace ? referenceForModel(ResourceQuotaModel) : referenceForModel(ClusterResourceQuotaModel);
 
+const quotaScopes = Object.freeze({
+  'Terminating': {label: 'Terminating', description: 'Affects pods that have an active deadline. These pods usually include builds, deployers, and jobs.'},
+  'NotTerminating': {label: 'Not Terminating', description: 'Affects pods that do not have an active deadline. These pods usually include your applications.'},
+  'BestEffort': {label: 'Best Effort', description: 'Affects pods that do not have resource limits set. These pods have a best effort quality of service.'},
+  'NotBestEffort': {label: 'Not Best Effort', description: 'Affects pods that have at least one resource limit set. These pods do not have a best effort quality of service.'},
+});
+
 const getResourceTypes = (quota) => {
   const specHard = _.get(quota, 'spec.hard');
   return _.keys(specHard).sort();
```

Comment:
Sorry, can you elaborate? -thanks

Reply:
Arrays are really inefficient to check if the item exists since you have to iterate over the entire array. Using `Set` is constant time with `mySet.has('foo')`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
I realize you need to sort, but the places that use `_.includes` should use a Set